### PR TITLE
Update Admin Notify

### PR DESCRIPTION
**Pull request by yesszus**
Add lang codes.

### CODE REVIEW

**Comment by yesszus** [comment]
@CaMer0n  pls. review this.

File: e107_admin/notify.php [hunk 1]
```diff
@@ -313,7 +313,7 @@ function render_event($id, $description, $include='', $legacy = 0)
 		$tp = e107::getParser();
 		$frm = e107::getForm();
 		$uc = e107::getUserClass();
-		$uc->fixed_classes['email'] = 'Email Address =>';
+		$uc->fixed_classes['email'] = LAN_EMAIL;
```
Comment (CaMer0n) [inline e107_admin/notify.php]: We should add a font-awesome arrow in here, if possible.

File: e107_admin/notify.php [hunk 2]
```diff
@@ -144,7 +144,7 @@ function init()
 
 									$data = e107::callMethod($val."_notify", 'config');
 
-									$config_category = str_replace("_menu","",ucfirst($val))." Events";
+									$config_category = str_replace("_menu","",ucfirst($val))."&nbsp;".LAN_NOTIFY_01;
```
Comment (CaMer0n) [inline e107_admin/notify.php]: No combining.. we should use "[x] Events" here.

File: e107_languages/English/admin/lan_notify.php [hunk 3]
```diff
@@ -18,15 +22,21 @@
 define("NU_LAN_3", "User account verification");
 define("NU_LAN_4", "User login");
 define("NU_LAN_5", "User logout");
+define("NU_LAN_6", "User social login");
+define("NU_LAN_7", "User social signup");
+define("NU_LAN_8", "User views profile");
+define("NU_LAN_9", "User edits profile");
```
Comment (CaMer0n) [inline e107_languages/English/admin/lan_notify.php]: This all looks great, thank you.

**Comment by yesszus** [comment]
@CaMer0n updated this.

**Pull request closed without merging**